docs/sphinx/compiling/general.rst

```diff
@@ -33,16 +33,99 @@ Required for some options
 
 * `libevent <http://libevent.org/>`_
 * `DPDK dpdk-16.11.9 <http://dpdk.org/>`_ (see :ref:`dpdk`), requires hugepage support
-* `AMD OpenCL drivers <http://support.amd.com/en-us/download/linux>`_ and `SDK <http://developer.amd.com/amd-accelerated-parallel-processing-app-sdk/>`_
-* `AMD ROCm <https://github.com/RadeonOpenCompute/ROCm>`_
-* `HDF5 <https://www.hdfgroup.org/HDF5/>`_ and branch ``extensible-datasets`` of jrs65's fork of `HighFive <https://github.com/jrs65/HighFive>`_
-* `Bitshuffle <https://github.com/kiyo-masui/bitshuffle>`_
-* `h5py <http://docs.h5py.org/en/latest/build.html#source-installation>`_
+* `AMD ROCm <https://github.com/RadeonOpenCompute/ROCm>`_ (this also installs AMD OpenCL drivers and SDK)
+* `HDF5 <https://www.hdfgroup.org/HDF5/>`_:: 
+
+    sudo apt-get install libhdf5-serial-dev
+
+* Branch ``extensible-datasets`` of jrs65's fork of `HighFive <https://github.com/jrs65/HighFive>`_
+* Clone HighFive in ``/opt`` with::
+
+    git clone --single-branch --branch extensible-datasets https://github.com/jrs65/HighFive.git
+
+* `h5py <http://docs.h5py.org/en/latest/build.html#source-installation>`_ (Install from source for bitshuffle)::
+
+    python setup.py configure --hdf5=/usr/lib/x86_64-linux-gnu/hdf5/serial/ --hdf5-version=1.10.0
+    python setup.py install
+
+* `Bitshuffle <https://github.com/kiyo-masui/bitshuffle>`_ (Needs to be installed after h5py)::
+
+    export HDF5_PLUGIN_PATH=/usr/local/hdf5/lib/plugin
+    python setup.py install --h5plugin --h5plugin-dir=/usr/local/hdf5/lib/plugin
+
 * `fftw3 <http://www.fftw.org/>`_
 * `libairspy <https://github.com/airspy/airspyone_host/tree/master/libairspy>`_
 * `OpenBLAS <http://www.openblas.net/>`_ (see :ref:`openblas`)
+* `LAPACK <http://www.netlib.org/lapack/>`_ (Should automatically be installed with OpenBLAS if a Fortran compiler is present on your system)
+* `Blaze <https://bitbucket.org/blaze-lib/blaze/src/master/>`_
+  * Download and unpack into `/opt`
 * OpenSSL (Package is called ``libssl-dev`` on Ubuntu).
-* `pytest-cpp <https://github.com/pytest-dev/pytest-cpp>`_
+* For python packages to be installed for all::
+
+    sudo su
+    cd ~
+    pip3 install python-module
```

Comment:
This seems a little out of place here.  Maybe make this a footnote and reference it when suggesting `pip` commands below?